src/main/java/edu/utdallas/davisbase/parser/Parser.java

```diff
@@ -15,13 +34,203 @@ public Parser(ParserConfiguration configuration) {
 
   /**
    * @param statement a single complete statement to parse
-   * @return the {@link Ast} representation of <code>statement</code>
+   * @return the {@link CommandRepresentation} representation of <code>statement</code>
    * @throws ParseException if <code>statement</code> is not a single complete statement that is
    *                        both lexically and syntactically correct
    */
-  public Ast parse(String statement) throws ParseException {
-    // TODO Implement Parser.parse(String)
-    throw new NotImplementedException();
+  public CommandRepresentation parse(String statement) throws ParseException {
+    try {
+      if(statement.trim().replaceAll("\\s+", " ").equalsIgnoreCase(new ExitCommandRepresentation().getFullCommand())){
```

Comment:
What if there's whitespace between "EXIT" and the semicolon? E.g. `EXIT ;` -> `EXIT ;` != `EXIT;`

What if there's whitespace before "EXIT"? E.g. `\nEXIT;` ->&nbsp;` EXIT;` != `EXIT;`

What if there's whitespace after the semicolon? E.g. `EXIT;\n` -> `EXIT; `&nbsp;!= `EXIT;`

```suggestion
      if (Pattern.matches("(?i)\\s*EXIT\\s*;\\s*", statement)) {
```

Where `Pattern` is `java.util.regex.Pattern`.